app/src/main/res/layout/register_screen.xml

```diff
@@ -8,7 +9,117 @@
             type="ip.team13.petowner.ui.register.RegisterViewModel" />
     </data>
 
-    <androidx.constraintlayout.widget.ConstraintLayout
+    <ScrollView
         android:layout_width="match_parent"
-        android:layout_height="match_parent" />
+        android:layout_height="match_parent"
+        android:fillViewport="true">
+
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <ImageView
+                android:id="@+id/civ_logo"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginTop="64dp"
+                android:scaleX="3"
+                android:scaleY="3"
```

Comment:
If you tried to achieve an image ratio by using these attributes, you could check out the `ConstraintLayout` ratio attribute